source/_integrations/yalexs_ble.markdown

```diff
@@ -0,0 +1,47 @@
+---
+title: Yale Access Bluetooth
+description: Instructions on how to integrate Yale Access Bluetooth devices into Home Assistant.
+ha_category:
+  - Lock
+ha_bluetooth: true
+ha_release: 2022.9
+ha_iot_class: Local Push
+ha_codeowners:
+  - '@bdraco'
+ha_domain: yalexs_ble
+ha_config_flow: true
+ha_platforms:
+  - lock
+ha_integration_type: integration
+---
+
+Integrates [Yale Access](https://www.yalehome.com/us/en/products/smart-technology/yale-access) Bluetooth devices into Home Assistant.
+
+{% include integrations/config_flow.md %}
+
+The Yale Access Bluetooth integration will automatically discover devices once the [Bluetooth](/integrations/bluetooth) integration is enabled and functional.
+
+## Supported devices
+
+Devices must have the Yale Access module installed.
+
+- YRD216
+- YRL216
+- YRD226
+- YRL226
+- YRD256
+
+## Obtaining the offline key
+
+The offline key and slot number are required to operate the lock. These credentials reside in the Yale Access app storage on iOS or Android devices with owner access to the lock.
```

Comment:
The next steps will pull the keys from the cloud without all this but thats not in the first PR.